.github/workflows/lint.yml

```diff
@@ -0,0 +1,61 @@
+name: Lint
+
+on: [push]
+
+jobs:
+
+  # We can't use VALIDATE_GO from super linter because of this issue:
+  # https://github.com/github/super-linter/issues/143
+  go-lint:
+    name: Lint go
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.43.0
+          args: --timeout 5m0s
+
+  md-link-check:
+    name: Check markdown links
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Check if Markdown links are valid
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with: 
+          config-file: '.github/linters/mlc_config.json'
+          folder-path: "."
+
+  super-lint:
+    name: Run super linter
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Super-Linter
+      uses: github/super-linter/slim@v4
+      env:
+        VALIDATE_GITLEAKS: true
+        VALIDATE_MARKDOWN: true
```

Comment:
Does this pick up / use the current config for Markdown? I'd ideally want to turn this to false and execute/handle Markdown separately. What I'm thinking of is also that Markdown linting is needed in completely non-code repos such as https://github.com/cheqd/cheqd-governance and I want to keep it consistent across all repos (there's no point running super linter for that repo)